Sources/Networking/Operations/PostRedeemWebPurchaseOperation.swift

```diff
@@ -0,0 +1,146 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  PostReceiptDataOperation.swift
+//
+//  Created by Joshua Liebowitz on 11/18/21.
```

Comment:
lol pretty sure this isn't right unless Josh was looking suuuper far ahead in 2021 and kept this file a secret for years